cmd/build-extensions-container.go

```diff
@@ -68,7 +68,7 @@ func buildExtensionContainer() error {
 	metapath := filepath.Join(buildPath, "meta.json")
```

Comment:
instead of reading in the `meta.json` here and defining a new `var cosaBuild cosa.Build` below, could we just use the `lastBuild` that we already have from the `cosa.ReadBuild()` call at the beginning of this function?

Reply:
if we do do that I wouldn't mind renaming `lastBuild` to `cosaBuild` above and using that variable name.

Reply:
Thanks for implementing this. I do realize now though that there is a lot more time that passes from the time we read the build (and current `meta.json`) and we write it out. I think that's OK, just wanted to call it out in case this raises any alarm bells with @jlebon.